internal/transform/transaction.go

```diff
@@ -216,6 +236,36 @@ func TransformTransaction(transaction ingest.LedgerTransaction, lhe xdr.LedgerHe
 	return transformedTransaction, nil
 }
 
+func getAccountBalanceFromLedgerEntryChanges(changes xdr.LedgerEntryChanges, sourceAccountAddress string) (int64, int64) {
+	var accountBalanceStart int64
+	var accountBalanceEnd int64
+
+	for _, change := range changes {
+		switch change.Type {
+		case xdr.LedgerEntryChangeTypeLedgerEntryUpdated:
+			accountEntry, ok := change.Updated.Data.GetAccount()
+			if !ok {
+				continue
+			}
+
+			if accountEntry.AccountId.Address() == sourceAccountAddress {
+				accountBalanceEnd = int64(accountEntry.Balance)
+			}
+		case xdr.LedgerEntryChangeTypeLedgerEntryState:
+			accountEntry, ok := change.State.Data.GetAccount()
+			if !ok {
+				continue
+			}
+
+			if accountEntry.AccountId.Address() == sourceAccountAddress {
+				accountBalanceStart = int64(accountEntry.Balance)
```

Comment:
will this logic work with fee bump transactions as well?

Reply:
None of the above logic runs for a fee bump transaction. Right now it only looks at V1 transactions which can have [sorobanData](https://github.com/stellar/stellar-etl/pull/231/files/213cb28a5564ad5b0d02f48d2aae09ffef3b043b#diff-c26dd0484eb96ce222a4019ab0809b202fd4b82ded179f0806674123667fd419R145)

Right now the fee for fee bump transactions is saved in `max_fee`. Do these fees need some breakdown as well?

Reply:
Updated to handle fee bump transactions as well. Although there aren't any in mainnet nor testnet so far